docs/supported-versions.md

```diff
@@ -1,9 +1,22 @@
 # Supported versions
 
-Starting with version 8.0.34-29, Percona XtraBackup supports backing up databases compatible with MySQL 8.0.34 and later. Specifically, Percona XtraBackup {{release}} supports taking backups of the following MySQL-compatible databases:
+Percona XtraBackup 8.0.34-29 supports backing up databases compatible with server version 8.0.34 and later. Percona XtraBackup 8.0.34-29 can create backups of older server versions if specific conditions are met. This capability is useful for maintaining compatibility and ensuring backups can be performed across different server versions. You might need to override server version check and set specific options, such as `lock-ddl=ON`.
+
+Specifically, Percona XtraBackup {{release}} supports taking backups of the following MySQL-compatible databases:
 
 * Percona Server for MySQL 8.0.34, and MySQL 8.0.34
 
 * Percona Server for MySQL 8.0.35, and MySQL 8.0.35
 
 * Percona Server for MySQL 8.0.36 and MySQL 8.0.36 (and any later versions in the 8.0 series)
+
+
```

Comment:
From here, we can provide the link to the version comparison/reference page and ask the user to look.

Reply:
We do - it's in the additional resources.

Reply:
I mean after the  line, "Percona Server for MySQL 8.0.36 and MySQL 8.0.36"

Please add some text like, "For additional information on the version compatibility, Please check here [a link to  Server version and backup version comparison[¶](https://percona-xtrabackup-docs-8-0-pr-302.onrender.com/server-backup-version-comparison.html#server-version-and-backup-version-comparison)

Reply:
Done

Reply:
I dont see it @patrickbirch